docs/release-notes.md

```diff
@@ -1,37 +1,3 @@
-**Trino Gateway documentation**
```

Comment:
mkdocs blames the wrong link in this file:
```
Doc file 'release-notes.md' contains a link '#4', but there is no such anchor on this page.
```

Reply:
Fixed .. I did not see that error .. did it show on the console when browsing to the page only?